rclcpp_lifecycle/src/lifecycle_node_interface_impl.hpp

```diff
@@ -425,12 +425,11 @@ class LifecycleNode::LifecycleNodeInterfaceImpl
       auto callback = it->second;
       try {
         cb_success = callback(State(previous_state));
-      } catch (const std::exception &) {
+      } catch (const std::exception & e) {
         // TODO(karsten1987): Windows CI doesn't let me print the msg here
         // the todo is to forward the exception to the on_error callback
-        // RCUTILS_LOG_ERROR("Caught exception in callback for transition %d\n",
-        //  it->first);
-        // RCUTILS_LOG_ERROR("Original error msg: %s\n", e.what());
+        RCUTILS_LOG_ERROR("Caught exception in callback for transition %d\n", it->first);
```

Comment:
I'm not sure how the `\n` is handled on Windows during compilation/runtime (since newlines are typically represented by `\r\n`), perhaps this is the cause of the build warning. In any case, we can drop the `\n` character from both log statements since (I think) RCUTILS_LOG_ERROR appends a newline.

Reply:
Okay, I have removed the \n characters.